renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "rebaseWhen": "conflicted",
+  "gitAuthor": "Renovate <renovate@oztechan.com>",
+  "commitBody": "Co-authored-by: Mustafa Ozhan <mr.mustafa.ozhan@gmail.com>",
```

Comment:
❓ Why the email and name should be appended to the commit body?

Reply:
I actually moved this from CCC, I can add you also if you want, it will add my account as co-author for the commits that renovate makes: This will have make me as one of the author and the 3rd party tools that we use (Codecov, Codacy) will scan our code since commits are partially from a known author.

Reply:
Did not know this :d No worries, if one email would suffice to not have the renovate commits not scanned then all good 👍